app/helpers/sufia/sufia_helper_behavior.rb

```diff
@@ -70,42 +72,52 @@ def number_of_deposits(user)
       ActiveFedora::Base.where(DepositSearchBuilder.depositor_field => user.user_key).count
     end
 
-    def link_to_facet(field, field_string)
-      path = search_action_path(search_state.add_facet_params_and_redirect(field_string, field))
-      link_to(field, path)
+    # @param item [Object]
+    # @param field [String]
+    # @return [ActiveSupport::SafeBuffer] the html_safe link
+    def link_to_facet(item, field)
+      path = search_action_path(search_state.add_facet_params_and_redirect(field, item))
+      link_to(item, path)
     end
 
-    # @param values [Array] The values to display
-    # @param solr_field [String] The name of the solr field to link to without its suffix (:facetable)
-    # @param empty_message [String] ('No value entered') The message to display if no values are passed in.
-    # @param separator [String] (', ') The value to join with.
-    def link_to_facet_list(values, solr_field, empty_message = "No value entered", separator = ", ")
+    # @param values [Array{String}] strings to display
+    # @param solr_field [String] name of the solr field to link to, without its suffix (:facetable)
+    # @param empty_message [String] message to display if no values are passed in
+    # @param separator [String] value to join with
+    # @return [ActiveSupport::SafeBuffer] the html_safe link
+    def link_to_facet_list(values, solr_field, empty_message = "No value entered".html_safe, separator = ", ")
       return empty_message if values.blank?
```

Comment:
Any reason not to move the `#html_safe` call here (vs. in the method signature)?